src/site/_includes/components/EventTable.js

```diff
@@ -41,17 +42,34 @@ module.exports = (event, authorsCollection) => {
     }
   }
 
+  const slugs = {};
+  const slugForTitle = (title) => {
+    // Find a slug for this title, but prevent duplicate IDs.
+    const base = slugify(title, {lower: true, strict: true, remove: /'/});
+    let id = base;
+    let suffix = 0;
+    while (id in slugs) {
+      id = base + ++suffix;
+    }
+    slugs[id] = title;
+    return id;
+  };
+
   const renderSession = ({speaker, title}) => {
     // Always pass an Array of author IDs.
     const authors = typeof speaker === 'string' ? [speaker] : speaker;
 
+    const id = slugForTitle(title);
+
     return html`
-      <tr>
-        <td class="w-event-schedule__speaker">
+      <div class="w-event-schedule__row" data-session-id=${id}>
+        <div class="w-event-schedule__cell w-event-schedule__speaker">
           ${AuthorsDate({authors}, authorsCollection)}
-        </td>
-        <td class="w-event-schedule__session">${title}</td>
-      </tr>
+        </div>
+        <div class="w-event-schedule__cell w-event-schedule__session">
+          <a class="w-event-schedule__open" href="#${id}">${title}</a>
```

Comment:
I dont' think we want the default blue styling for these links. I think the links should be black and hovering anywhere in the row should trigger the hover effect.